python/ray/autoscaler/v2/scheduler.py

```diff
@@ -100,11 +99,24 @@ class SchedulingNodeStatus(Enum):
 
     # The node is to be launched.
     TO_LAUNCH = "TO_LAUNCH"
-    # The node is pending, i.e. there's already an autoscaler instance being launched
-    PENDING = "PENDING"
-    # The node is running.
+    # Ray is pending on the node.
+    RAY_PENDING = "PENDING"
+    # ray is already running.
     RUNNING = "RUNNING"
 
+    @staticmethod
+    def is_ray_pending(instance: Instance) -> bool:
+        """
+        Check if the instance is pending on ray.
+        """
+        return instance.status in [
```

Comment:
I remember InstanceUtil has the same function?

Reply:
right - modified.